typings.json

```diff
@@ -5,6 +5,7 @@
     "tape": "github:DefinitelyTyped/DefinitelyTyped/tape/tape.d.ts#221df9ad820d0b2aa010d328c4bab06c8d69ac1f"
   },
   "dependencies": {
+    "chai": "registry:npm/chai#3.5.0+20160328084239",
```

Comment:
Wait, nevermind. I see what's going on. Ignore this. I'm going to undo this commit.